### DISCUSSION TITLE
allow multiple error blocks from a single path

### DISCUSSION THREAD

**Discussion by savaki** [graph-gophers/graphql-go#623] [Ideas]
Currently, only a single error block will be generated from a given path.  With the advent of the `Unwrap() []error` interface that builtin methods like `errors.Join` provide, graphql should treat this as multiple errors for the same path

**Comment by pavelnikolov** [comment]
This is against the GraphQL spec, which clearly states:

> If the field returns null because of a field error which has already been added to the "errors" list in the response, the "errors" list must not be further affected. That is, only one error should be added to the errors list per field.

https://spec.graphql.org/October2021/#sel-EANRNDLAACNAn7V
